packages/zowe-explorer/i18n/sample/src/job/utils.i18n.json

```diff
@@ -3,5 +3,7 @@
   "jobs.sortByDateSubmitted": "$(calendar) Date Submitted",
   "jobs.sortByName": "$(case-sensitive) Job Name",
   "jobs.sortByReturnCode": "$(symbol-numeric) Return Code",
-  "setSortDirection": "$(fold) Sort Direction"
+  "setSortDirection": "$(fold) Sort Direction",
+  "filterJobs.quickpick.message": "Go to Local Filtering",
+  "filterJobs.clearProfileFilter": "$(clear-all) Clear filter for profile"
```

Comment:
I noticed that a new string was added to the jobs tree for clearing the profile filter. Since [this string](https://github.com/zowe/vscode-extension-for-zowe/blob/68c9167d81e51b4a5ed5da88654cbe55676a9f77/packages/zowe-explorer/i18n/sample/src/dataset/DatasetTree.i18n.json#L36) is the same, do you mind changing the string to something generic like `filter.clearForProfile` and then using it for both the Data Sets and Jobs filtering features?

Not my primary request for changes, but would avoid duplication of localized strings.